rc-virtual-list 3.10.0  problems with the last update

`#13 8.176 ./node_modules/rc-virtual-list/lib/List.d.ts:5:15
#13 8.176 Type error: ',' expected.
#13 8.176
#13 8.176   3 | import type { ScrollBarDirectionType } from './ScrollBar';
#13 8.176   4 | import type { RenderFunc, ExtraRenderInfo } from './interface';
#13 8.176 > 5 | import { type ScrollPos, type ScrollTarget } from './hooks/useScrollTo';
#13 8.176     |               ^
#13 8.176   6 | export interface ScrollInfo {
#13 8.176   7 |     x: number;
#13 8.176   8 |     y: number;
#13 8.204 npm notice`


Hello, today after the update I've been suffering from this problem only when the application builds, I don't use the lib directly as it depended on antd if I'm not mistaken. Does anyone know how to fix?

We are running into the exact same error when running on Github Actions. Any assistance would be great.

```
#19 [15/16] RUN yarn workspace shared-library compile
#19 6.375 ../../node_modules/rc-virtual-list/lib/List.d.ts(5,15): error TS1005: ',' expected.
#19 6.377 ../../node_modules/rc-virtual-list/lib/List.d.ts(5,31): error TS1005: ',' expected.
#19 ERROR: process "/bin/sh -c yarn workspace shared-library compile" did not complete successfully: exit code: 2
```

--- a/src/List.tsx
+++ b/src/List.tsx
@@ -11,7 +11,8 @@ import ScrollBar from './ScrollBar';
 import type { RenderFunc, SharedConfig, GetKey, ExtraRenderInfo } from './interface';
 import useChildren from './hooks/useChildren';
 import useHeights from './hooks/useHeights';
-import useScrollTo, { type ScrollPos, type ScrollTarget } from './hooks/useScrollTo';
+import useScrollTo from './hooks/useScrollTo';
+import type { ScrollPos, ScrollTarget } from './hooks/useScrollTo';
 import useDiffItem from './hooks/useDiffItem';
 import useFrameWheel from './hooks/useFrameWheel';
 import useMobileTouchMove from './hooks/useMobileTouchMove';
@@ -255,14 +256,14 @@ export function RawList<T>(props: ListProps<T>, ref: React.Ref<ListRef>) {
   const verticalScrollBarRef = useRef<ScrollBarRef>();
   const horizontalScrollBarRef = useRef<ScrollBarRef>();
 
-  const horizontalScrollBarSpinSize = React.useMemo(
-    () => getSpinSize(size.width, scrollWidth),
-    [size.width, scrollWidth],
-  );
-  const verticalScrollBarSpinSize = React.useMemo(
-    () => getSpinSize(size.height, scrollHeight),
-    [size.height, scrollHeight],
-  );
+  const horizontalScrollBarSpinSize = React.useMemo(() => getSpinSize(size.width, scrollWidth), [
+    size.width,
+    scrollWidth,
+  ]);
+  const verticalScrollBarSpinSize = React.useMemo(() => getSpinSize(size.height, scrollHeight), [
+    size.height,
+    scrollHeight,
+  ]);
 
   // =============================== In Range ===============================
   const maxScrollHeight = scrollHeight - height;

